includes/class-sensei-modules.php

```diff
@@ -1243,19 +1243,60 @@ public function course_columns( $columns = array() ) {
 	 */
 	public function course_column_content( $column = '', $course_id = 0 ) {
 		if ( 'modules' === $column ) {
-			$module_links = $this->get_module_links( $course_id );
+			$this->output_course_modules_column( $course_id );
+		}
+	}
+
+	/**
+	 * Output the course modules column HTML.
+	 *
+	 * @since 4.0.0
+	 *
+	 * @param int $course_id
+	 * @return void
+	 */
+	private function output_course_modules_column( int $course_id ) {
+		$module_links = $this->get_module_links( $course_id );
 
-			if ( $module_links ) {
-				echo implode( ', ', $module_links ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Output is escaped in the array.
+		if ( ! $module_links ) {
+			return;
+		}
 
-				// Output the edit modules order link.
-				echo sprintf(
-					'<a class="button-link" href="%s">%s</a>',
-					esc_url( admin_url( 'edit.php?post_type=course&page=module-order&course_id=' . intval( $course_id ) ) ),
-					esc_html__( 'Order modules', 'sensei-lms' )
-				);
+		$links_count     = count( $module_links );
+		$max_links_count = apply_filters( 'sensei_modules_column_max_links_count', 3 );
```

Comment:
Probably we should add some doc-block for the new filter?
Here is an example: https://github.com/Automattic/sensei/blob/634743e50689f5493061fa04d859cdf7dd1612a7/includes/class-sensei-quiz.php#L75-L87

Reply:
Thanks! Doc block added in ebde6f2d41e9f0906dbd596cbf60a613ef740438.